Rewrite docs (#369)
DevExpress/DevExtreme.AspNet.Data: README.md

[![npm](https://img.shields.io/npm/v/devextreme-aspnet-data.svg?maxAge=43200)](https://www.npmjs.com/package/devextreme-aspnet-data)
[![npm nojquery](https://img.shields.io/npm/v/devextreme-aspnet-data-nojquery.svg?maxAge=43200&label=npm+nojquery)](https://www.npmjs.com/package/devextreme-aspnet-data-nojquery)

> This library enables [DevExtreme client-side widgets](https://js.devexpress.com) to consume data from ASP.NET controllers and handlers. A widget communicates with the server in the following manner: 

Reviewer: Why 'consume'?

Author: This word is typically used in such cases. You can search "consume data from" in Internet.

Reviewer: verb: consume; 3rd person present: consumes; past tense: consumed; past participle: consumed; gerund or present participle: consuming

1) eat, drink, or ingest (food or drink).
"people consume a good deal of sugar in drinks"
synonyms: eat, eat up, devour, ingest, swallow, gobble, gobble up, wolf down, gorge oneself on, feast on

2) (of a fire) completely destroy.
"the fire spread rapidly, consuming many homes"
synonyms: destroy, demolish, lay waste, wipe out, annihilate, devastate

3) use up (a resource).
"this process consumes enormous amounts of energy"
synonyms: use, use up, utilize, expend

3) buy (goods or services).
"accounting provides measures of the economic goods and services consumed"

Reviewer: I'm familiar with the context in which 'consume data' is typically used - it is used in the same way as no.3 in that list. So do you mean to say that it uses up all the data from ASP.NET controllers and handlers?

Reviewer 2: Hello @dirk-pieterse. We use 'consume' in the same meaning as this Google search reveals: https://google.com/search?q=consume+data,+web+service. This is an idiomatic phrase. Typically used in the context of web services and APIs. Instead of 'consume', we can write 'perfrom CRUD operations using/via ASP.NET controllers and handlers'.

Reviewer: idiomatic phrases are not suitable for technical texts because they have multiple interpretations (both figurative and literal) - they are ambiguous.

I would recommend: perform CRUD operations via ASP.NET controllers and handlers

- The widget sends data loading options (filtering, grouping, sorting, etc) to the server.
- The server processes data according to these options and then sends processed data back to the widget.

> In this way, you delegate all intensive data operations to the server.

Reviewer: "In this way"? Which way?

Author: Can we rewrite "Thus you delegate all ...." ?

Reviewer: You cannot start a sentence with 'Thus'

Author: Rewritten: 

> As you can see, all intensive data operations are delegated to the server.

or your variant?

Reviewer 2: @dirk-pieterse Please suggest how we can build a better phrase. We summarize that things described above effectively mean ('in other words') that 'you delegate...'

Reviewer: This library enables [DevExtreme client-side widgets](https://js.devexpress.com) to perform CRUD operations via ASP.NET controllers and handlers and allows you to delegate all data-intensive operations to the server:

- the widget sends data loading options (filtering, grouping, sorting, etc) to the server
- the server processes data according to these options
- the processed data is sent back to the widget

OR

This library enables [DevExtreme client-side widgets](https://js.devexpress.com) to perform CRUD operations via ASP.NET controllers and handlers and allows you to delegate all data-intensive operations to the server. To do this,  the widget sends data loading options (filtering, grouping, sorting, etc) to the server, the server processes data according to these options, and the processed data is sent back to the widget.

Can be used with:

* [DevExtreme client-side widgets](https://js.devexpress.com)
* [DevExtreme-based ASP.NET Core controls](https://docs.devexpress.com/AspNetCore/400263)
* [DevExtreme ASP.NET MVC 5 controls](https://docs.devexpress.com/DevExtremeAspNetMvc/400943/)

## Installation and Configuration

> `DevExtreme.AspNet.Data` consists of the server-side and client-side parts. Learn how to install and configure both the parts from the following topics:

Reviewer: ```suggestion
`DevExtreme.AspNet.Data` consists of server-side and client-side parts. The following topics explain how to install and configure these parts:
```

- [Server Side Configuration](docs/server-side-configuration.md)
- [Client Side with jQuery](docs/client-side-with-jquery.md)
- [Client Side without jQuery (Angular, etc.)](docs/client-side-without-jquery.md)

## CI Builds

> We recommend to use [release builds](https://github.com/DevExpress/DevExtreme.AspNet.Data/releases). However, you can use [CI builds](docs/using-ci-builds.md) to get urgent bug fixes or to test not-yet-released functionality.

Reviewer: ```suggestion
We recommend that you use [release builds](https://github.com/DevExpress/DevExtreme.AspNet.Data/releases). However, you can also use [CI builds](docs/using-ci-builds.md) to get urgent bug fixes or to test unreleased functionality.
```
